README.md

```diff
@@ -1298,5 +1303,147 @@ export default CustomGrid
 
 ```
 
+### Partial charts
+Here's another example of how the `extras` property can be used to create highly customizable charts. You can use it to create overlays and, with the svg's clipPath capability, create charts that partially have a different look-and-feel. See the example below.
+
+![Custom grid](https://raw.githubusercontent.com/jesperlekland/react-native-svg-charts/master/screenshots/partial-charts.png)
+
+### Example
+```javascript
+import React from 'react'
+import { View, Text } from 'react-native'
+import { 
+    LinearGradient, 
+    Stop, 
+    Defs, 
+    ClipPath, 
+    Rect, 
+    Path, 
+} from 'react-native-svg'
+import { LineChart, AreaChart } from 'react-native-svg-charts'
+import * as shape from 'd3-shape'
+
+class PartialChartExample extends React.PureComponent {
+    render() {
+        const data = [ 50, 10, 40, 95, -4, -24, 85, 91, 35, 53, -53, 24, 50, -20, -80 ]
+
+        const indexToClipFrom = 10
+
+        const ExtraDefs = ({ x, width }) => (
+            <Defs key={'defs-1'}>
+                <LinearGradient id={ 'gradient' } x1={ '0%' } y={ '0%' } x2={ '0%' } y2={ '100%' }>
+                    <Stop offset={ '0%' } stopColor={ 'rgb(134, 65, 244)' } stopOpacity={ 0.8 }/>
+                    <Stop offset={ '100%' } stopColor={ 'rgb(134, 65, 244)' } stopOpacity={ 0.2 }/>
+                </LinearGradient>
+                <ClipPath id={'clip-path-1'} key={'0'}>
+                    <Rect x={x(indexToClipFrom)} y={'0'} width={width - x(indexToClipFrom)} height={'100%'} />
+                </ClipPath>
+                <ClipPath id="clip-path-2" key={'1'}>
+                    <Rect x={'0'} y={'0'} width={x(indexToClipFrom)} height={'100%'} />
+                </ClipPath>
+            </Defs>
+        )
+
+        // Line extras:
+        const OverlayLine = ({ line }) => (
+            <Path
+                key={'line-1'}
+                d={line}
+                stroke={ 'rgb(134, 65, 244)' }
+                strokeWidth={ 2 }
+                fill={ 'none' }
+                clipPath={ 'url(#clip-path-2)' }
+            />
+        )
+
+        const OverlayShadowLine = ({ shadow }) => (
```

Comment:
I did some digging. We can actually remove the shadow path all together, don't render it in the chart and just use this api with the exact same path as `line` but with a `y` prop of e.g "2", works like a charm and is super modular.